src/SVGSkin.js

```diff
@@ -1,9 +1,9 @@
 const twgl = require('twgl.js');
 
+const Silhouette = require('./Silhouette');
 const Skin = require('./Skin');
-const SvgRenderer = require('./svg-quirks-mode/svg-renderer');
 
-const Silhouette = require('./Silhouette');
+import SvgRenderer from 'scratch-svg-renderer';
```

Comment:
I think we should stick with using require if we are using it elsewhere and not switch to import?

Reply:
👍 the `scratch-svg-renderer` was using `export default` so I used `import` here (otherwise the build would fail). However, I accidentally changed it to `module.exports = ...` when I merged from `scratch-render` into `scratch-svg-renderer` so... I guess I'll change these too ;)